toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -318,6 +321,22 @@ func findSourcePartitionHelper(source string,
 	return mountIdType, partition, partitionIndex, nil
 }
 
+func findExtendedSourcePartitionHelper(source string, partitions []diskutils.PartitionInfo,
```

Comment:
Can `findSourcePartitionHelper` be implemented using `findExtendedSourcePartitionHelper` to reduce duplicate code?